config/application.rb

```diff
@@ -1,12 +1,19 @@
 require_relative "boot"
 
 require "rails"
+# Pick the frameworks you want:
+# require "active_model/railtie"
+# require "active_job/railtie"
 require "active_record/railtie"
+# require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
 require "action_view/railtie"
 require "action_cable/engine"
```

Comment:
I can't see where actioncable is being used in this app. Could potentially be removed I guess.. if that's in scope of the upgrade..

Reply:
hmm true. I'm gonna take it out and see what happens